Symfony/CS/ConfigInterface.php

```diff
@@ -105,4 +105,20 @@ public function usingCache();
      * @return bool
      */
     public function usingLinter();
+
+    /**
+     * Sets the path to the cache file.
+     *
+     * @param string $cacheFile The path to the cache file
+     *
+     * @return ConfigInterface The same instance
```

Comment:
@keradus The method I am talking about is documented as NOT being chaninable. I'm saying this needs fixing either by making them all chainable, or the reverse. Sorry if I was unclear.